h2o-r/h2o-package/R/models.R

```diff
@@ -3064,5 +3064,31 @@ h2o.deepfeatures <- function(object, data, layer) {
   h2o.getFrame(dest_key)
 }
 
+setClass("H2OTree", representation(left_children="integer", right_children="integer", descriptions="character", model_name = "character",tree_number = "integer", tree_class="integer"))
+
+#' Fetchces a model's tree
+#'
+#' Extract the non-linear feature from an H2O data set using an H2O deep learning
+#' model.
+#' @param model Models with trees
+#' @param tree_number Number of tree in the model to fetch
+#' @param tree_class Class of the tree (if applicable)
+#' @return Returns an H2OTree object with detailed information about a tree.
+#' @export
+h2o.getModelTree <- function(model, tree_number, tree_class) {
```

Comment:
@Pscheidl run the name by #api-updates channel (cc: @ledell)

The output needs to be documented, it is not obviou what 

Please include an example of using this API, ideally there should be a small example in the docustring of this method and a demo similar to http://scikit-learn.org/stable/auto_examples/tree/plot_unveil_tree_structure.html - because your output structure is very similar you can adapt the example for R.

Reply:
Should I put the demo also in the docs ? I guess not, only making sure we understand each other. I'll change the docs to describe it much better of course.